libs/core/src/components/sage-checkbox/sage-checkbox.tsx

```diff
@@ -0,0 +1,106 @@
+import { Component, h, Prop, Host } from '@stencil/core';
+
+@Component({
+  tag: 'sage-checkbox',
+  styleUrl: 'sage-checkbox.scss',
+  shadow: true,
+})
+export class SageCheckbox {
+  /**
+   * Whether or not the checkbox is checked.
```

Comment:
Seeing inconsistencies here with some descriptions ending with a period and some not, can we unify?